use_cases/eluc/app/utils.py

```diff
@@ -265,17 +218,14 @@ def create_pie(data=pd.Series, type_context=True, year=2021) -> go.Figure:
         )
     )
 
+    # Remove the legend from the left plot so that we don't have 2
     if type_context:
         fig.update_layout(showlegend=False)
-        # To make up for the hidden legend
-        fig.update_layout(margin={"t": 50, "b": 50, "l": 50, "r": 50})
```

Comment:
Had to remove this code that added margins to the left pie chart because with the new size of the pie charts (since the extra crop sliders got removed so the chart is smaller) it's no longer necessary